### DISCUSSION TITLE
DolarBot - Ideas, sugerencias y opiniones

### DISCUSSION THREAD

**Discussion by guidospadavecchia** [guidospadavecchia/DolarBot#2] [Ideas]
## 👋 Hola!
  Utilizá este espacio para **expresar tu opinión**, **proponer ideas** y **hacer sugerencias** acerca del desarrollo de **DolarBot**.

**Comment by markito12** [comment]
Estaria bueno que pongas alertas de cuando el dolar llega a cierto valor o que avise cada X tiempo el valor del mismo. Excelente proyecto 😄

**Reply by guidospadavecchia** [reply]
Hola @markito12 !

Gracias por utilizar este espacio.

Mirá, eso que me comentás se me ocurrió hace unos meses, que cada persona o servidor pueda configurar alertas y que avise por DM a usuarios o envíe mensajes en cierto canal. La realidad es que aún no lo hice porque para eso, **DolarBot** tendría que utilizar una base de datos para persistir estas configuraciones. Hoy en día, **DolarBot** no es más que un intermediario entre los usuarios de Discord y las distintas fuentes de donde saca la información. Es decir, no está guardando ningún tipo de dato.  

Actualmente no estoy teniendo el tiempo para sacar mejoras tan rápido como hace unos meses, y esto llevaría unos cuantos días de desarrollo. No quiero prometerte nada pero creeme que lo tengo en cuenta para alguna versión futura.

**Reply by markito12** [reply]
No hay problema! 😉
